spec/service_models/miq_ae_service_miq_provision_spec.rb

```diff
@@ -110,13 +110,12 @@ def invoke_ae
   end
 
   it "#set_vm_notes" do
-    method   = "$evm.root['#{@ae_result_key}'] = $evm.root['miq_provision'].set_vm_notes"
+    method = "$evm.root['#{@ae_result_key}'] = $evm.root['miq_provision'].set_vm_notes('bite_me')"
     @ae_method.update(:data => method)
-
-    # %w{ template clone_to_vm clone_to_template }.each do |provision_type|
-    #  @miq_provision.update(:provision_type => provision_type)
-    #  invoke_ae.root(@ae_result_key).should == @miq_provision.provision_type
-    # end
+    expect(invoke_ae.root(@ae_result_key)).to be_truthy
+    expect(@miq_provision[:options][:vm_notes]).to be_nil
+    @miq_provision.reload
+    expect(@miq_provision[:options][:vm_notes]).to eq('bite_me')
```

Comment:
I prefer `@miq_provision.options` over the `[]` reference, but this seems to be consistent with other lines in the PR.  Keep it this way for the PR and we can clean up in a followup refactoring.